src/tests/ngraph_helpers/snippets_ngraph_functions/include/subgraph_simple.hpp

```diff
@@ -86,6 +100,23 @@ class EltwiseThreeInputsSinhFunction : public SnippetsFunctionBase {
 protected:
     std::shared_ptr<ov::Model> initOriginal() const override;
 };
+/// Eltwise graph with 10 inputs and 2 outputs.
+/// Needed to test for a max number of inputs+outputs allowed.
+// in1   in2   in3 ... in10
+// Sinh  Sinh  Sinh ...Sinh
```

Comment:
Did I get it right that Sinh on input is required to avoid "No subgraph after input limitation" and can be simplified after limitation is handled? If yes please leave corresponding to-dos.

Reply:
Exactly. Todo added.